imapclient/imapclient.py

```diff
@@ -268,6 +273,33 @@ def shutdown(self):
         """
         self._imap.shutdown()
 
+    def enable(self, *capabilities):
+        """Activate one or more server side capability extensions.
+
+        Most capabilities do not need to be enabled. This is only
+        required for extensions which introduce backwards incompatible
+        behaviour. Two capabilities which may require enable are
+        ``CONDSTORE`` and ``UTF8=ACCEPT``.
+
+        A list of the requested extensions that were successfully
+        enabled on the server is returned.
+
+        Once enabled each extension remains active until the IMAP
+        connection is closed.
+
+        See :rfc:`5161` for more details.
+        """
+        if self._imap.state != 'AUTH':
+            raise self.Error("ENABLE command illegal in state %s" % self._imap.state)
+
+        resp = self._raw_command_untagged(
+            b'ENABLE',
+            [to_bytes(c) for c in capabilities],
+            uid=False,
+            response_name='ENABLED',
+            unpack=True)
+        return resp.split()
```

Comment:
I was able to reproduce what you saw using Fastmail. Fastmail doesn't send an ENABLED response when nothing was enabled, whereas Gmail sends an empty ENABLED response.